README.rst

```diff
@@ -45,9 +45,9 @@ for detailed information on how to use **niflexlogger-automation**.
 
 .. _tests_section:
 
-Tests
+Automated Tests
 =====
-To run the tests, you must first configure FlexLogger to load the test
+To run the automated tests for the Python API, you must first configure FlexLogger to load the test
```

Comment:
A suggestion: add another sentence or two that introduces the concept/purpose of the python automated tests. "The Python API package contains a number of automated tests which should be used to validate API changes before submitting a pull request. If a pull request contains new API functionality, new automated tests that exercise the new functionality should be included with the pull request."